lumber_yard.rb

```diff
@@ -86,31 +91,43 @@
     username: params[:username],
     employee_type: params[:employee_type],
     }).valid?
-    @options = ModelCitizen::Messages.new
-    @success = false
-    @message = ModelCitizen::Messages.new.get_message(:invalid_employee)
-    erb :'employee/form'
+    flash[:employee_error] = ModelCitizen::Messages.new.get_message(:invalid_employee)
+    redirect '/employees/new'
   else
-    @options = ModelCitizen::Messages.new
-    @success = true
-    @message = ModelCitizen::Messages.new.get_message(:employee_success)
-    erb :'employee/form'
+    flash[:employee_success] = ModelCitizen::Messages.new.get_message(:employee_success)
+    redirect '/home/index'
   end
 end
 
-post '/clients/create' do
+post '/clients' do
   if !LumberYard::Client.new.create_client({
     name: params[:name],
     type: params[:type]
     }).valid?
-    @options = ModelCitizen::Messages.new
-    @success = false
-    @message = ModelCitizen::Messages.new.get_message(:invalid_client)
-    erb :'client/form'
+    flash[:client_error] = ModelCitizen::Messages.new.get_message(:invalid_client)
+    redirect '/clients/new'
   else
-    @options = ModelCitizen::Messages.new
-    @success = true
-    @message = ModelCitizen::Messages.new.get_message(:client_success)
-    erb :'client/form'
+    flash[:client_success] = ModelCitizen::Messages.new.get_message(:client_success)
+    redirect '/home/index'
   end
+end
+
+def set_session_data(employee)
```

Comment:
Also - storing this information session is a dangerous line to walk in terms of security. There are many tools available to manipulate session data, and if you store the employee type in the session, a normal employee could easily change their type to be an admin and get access to all sorts of features they shouldn't be able to see.

Again, for your app - I'm OK with this but we should add a story to the backlog to address this vulnerability.